src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -124,16 +125,18 @@ function ReportActionItemMessageEdit(props) {
     }, [isFocused]);
 
     useEffect(() => {
-        // For mobile Safari, updating the selection prop on an unfocused input will cause it to automatically gain focus
+        // For every platform execept Android, updating the selection prop on an unfocused input will cause it to automatically gain focus
```

Comment:
Could you compare it with the implementation in `ReportActionCompose.js`?

https://github.com/Expensify/App/blob/dd0d98fa6868bba60e81bfdf66e5cdf7d2adaa37/src/pages/home/report/ReportActionCompose.js#L231-L234

Maybe apart from the setDraft/setSelection we need to init the selection per platform too?

Reply:
As i can see in `ReportActionCompose.js` its called directly during initialisation of the state. And what we are trying to achieve is different as we want to call this after the comment state is initialised and also handle few of the browser related issues like `modal focus trap` etc.

Reply:
@robertKozik Tried your solution it works as we expected.